pinc/unicode.inc

```diff
@@ -170,3 +193,43 @@ function guess_string_encoding($text)
     # Give up and return ISO-8859-1
     return 'ISO-8859-1';
 }
+
+function make_title($codepoints)
+{
+    // make title from array of base codepoint and possible combining marks
+    $title = IntlChar::charName($codepoints[0]);
+    $count = count($codepoints);
+    if($count > 1)
+    {
+        $title .= " with " . IntlChar::charName($codepoints[1]);
+    }
+    for($index = 2; $index < $count; $index++)
+    {
+        $title .= " and " . IntlChar::charName($codepoints[$index]);
+    }
+    return $title;
+}
+
+function extended_to_hex($char)
```

Comment:
Is "extended" a wording in Unicode to represented combined characters? I feel like "combined" is a better description and how we've talked about it so far.

`combined_character_to_hex()` perhaps.

Reply:
I think you are right. 'Combining character sequence' seems to be the official phrase according to [http://www.unicode.org/versions/Unicode12.1.0/ch03.pdf#G30602](url).
I've changed this and all similar things.